src/integrationtests/java/com/aws/greengrass/integrationtests/extensions/IntegrationTestExtensionTest.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.integrationtests.extensions;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+@IntegrationTest
+public class IntegrationTestExtensionTest {
```

Comment:
running this test will eat a lot of time for no benefit won't it?

Reply:
yeah it's only beneficial for this PR to show that it plays nicely with ubuntu and windows github actions runners

Reply:
will disable it now since we have a test run, but keep the class there since it has good usage examples